portal/src/store/modules/materials.js

```diff
@@ -57,11 +57,17 @@ export default {
   },
   actions: {
     async getMaterial({ commit }, { id, params }) {
-      commit('SET_MATERIAL_LOADING', true)
-      const { data: material } = await axios.get(`materials/${id}/`, { params })
-      decodeAuthor(material)
-      commit('SET_MATERIAL', material)
-      commit('SET_MATERIAL_LOADING', false)
+      if (id) {
```

Comment:
Dit is wat mij betreft wat onduidelijk. Als je nu een lege Array doorgeeft als id, dan gaat hij dat proberen door te geven aan de request. Het kan niet veel kwaad in dit stukje code, maar ik zou zelf een isEmpty eromheen schrijven. Niet een blokker.

Reply:
✅